docs/other/troubleshooting-gratia.md

```diff
@@ -7,7 +7,11 @@ This document will help you troubleshoot problems with the Gratia Accounting, pa
 Gratia/GRACC: The Big Picture
 -----------------------
 
-Gratia is software used in OSG to gather accounting information. The information is collected from individual resources at a site, such as a Compute Entrypoint or a a submission host. The program that collects the data is called a "Gratia probe". The information is transferred to a GRACC server. Most sites will choose to send the accounting data to the central OSG Gratia server, but you can also use a Gratia server at your site (which can send forward the data to the central OSG Gratia server). Here is a diagram:
+Gratia is software used in OSG to gather accounting information.
+The information is collected from individual resources at a site, such as a Compute Entrypoint or a a submission host.
+The program that collects the data is called a "Gratia probe".
+The information is transferred to the central OSG GRACC server.
```

Comment:
```suggestion
The information is transferred to the [central OSG GRACC server](https://gracc.opensciencegrid.org/).
```

Reply:
I don't think this link would fit here.

- This link already appears later under "You can see the OSG's GRACC website at \[URL\]".
- As seen in the diagram right below this paragraph, this document makes a distinction between the "GRACC server" and the "GRACC website".  The text hear discusses the "server" but you are giving a link to the "website".

So i think adding this link is redundant and only serves to confuse things.

Reply:
Sounds good to me.